com.unity.cinemachine/Runtime/Behaviours/CinemachineConfiner2D.cs

```diff
@@ -101,19 +107,26 @@ void Reset()
             MaxWindowSize = -1;
         }
 
+        /// <summary>
+        /// Adjusts the confiner to the Cinemachine Camera's window size.
+        /// Call this when you change Orthographic Size or Field of View of the Cinemachine Camera's lens.
+        /// Calculating the new confiner is fast, but causes allocations.
+        /// </summary>
+        /// <remarks>
+        /// Most often, it is more efficient to have more Cinemachine Cameras with different lens settings
+        /// that have their own confiners and blend between them instead of changing
+        /// one Cinemachine Camera's lens and calling this over and over.
+        /// </remarks>
+        public void AdjustConfiner() => m_AdjustConfiner = true;
```

Comment:
Can we think of a better name?  "AdjustConfiner" says basically nothing about why you would want to invoke it, and as a user it would just make me feel nervous.

Reply:
InvalidateComputedConfiner?

I renamed it to that. Cleaned up; there is no need for a flag.